packages/compiler-core/src/utils.ts

```diff
@@ -59,8 +59,16 @@ export const walkJS: typeof walk = (ast, walker) => {
   return walk(ast, walker)
 }
 
+const reservedKeywords = new Set(
```

Comment:
It is already here: https://github.com/vuejs/vue-next/blob/master/packages/shared/src/globalsWhitelist.ts